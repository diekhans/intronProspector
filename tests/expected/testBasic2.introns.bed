chr10	5000666	5001513	sj0	1	-	5000666	5001513	64,128,64
chr10	5096577	5097433	sj1	1	-	5096577	5097433	64,128,64
chr10	5097550	5098801	sj2	1	-	5097550	5098801	64,128,64
chr10	5098879	5099326	sj3	1	+	5098879	5099326	64,128,64
chr10	5099449	5102100	sj4	1	+	5099449	5102100	64,128,64
chr10	5102650	5105594	sj5	1	+	5102650	5105594	64,128,64
chr10	5453560	5454264	sj6	41	+	5453560	5454264	64,128,64
chr10	5454522	5454947	sj7	55	-	5454522	5454947	64,128,64
chr10	5455118	5456086	sj8	45	+	5455118	5456086	64,128,64
chr10	5456273	5456587	sj9	46	-	5456273	5456587	64,128,64
