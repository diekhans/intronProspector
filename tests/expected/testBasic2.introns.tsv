chrom	intron_start	intron_end	strand	uniq_mapped_count	multi_mapped_count	unsure_mapped_count	max_left_overhang	max_right_overhang	confidence	splice_sites	name
chr10	5000666	5001513	-	0	1	0	81	20	0		sj0
chr10	5096577	5097433	-	0	1	0	72	29	0		sj1
chr10	5097550	5098801	-	0	1	0	85	16	0		sj2
chr10	5098879	5099326	+	1	0	0	80	17	0		sj3
chr10	5099449	5102100	+	0	1	0	15	85	0		sj4
chr10	5102650	5105594	+	0	1	0	75	26	0		sj5
chr10	5453560	5454264	+	41	0	0	68	86	4.03		sj6
chr10	5454522	5454947	-	53	2	0	87	91	4.45		sj7
chr10	5455118	5456086	+	45	0	0	85	90	4.75		sj8
chr10	5456273	5456587	-	46	0	0	86	79	4.69		sj9
